bigbluebutton-html5/imports/ui/components/connection-status/component.jsx

```
@@ -186,11 +148,42 @@ const ConnectionStatus = ({
+  }, []);
 
-      if (timeoutRef.current) {
-        clearTimeout(timeoutRef.current);
+  useEffect(() => {
+    const rttWorker = createRttWorker();
+    rttWorker.onmessage = (ev) => {
+      const { type } = ev.data || {};
+      switch (type) {
+        case 'rtt':
+          // eslint-disable-next-line no-case-declarations
+          const { rtt, serverEpochMsec } = ev.data || {};
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)